aikau/src/main/resources/alfresco/forms/controls/css/MultiSelect.css

```diff
@@ -0,0 +1,174 @@
+.alfresco-forms-controls-MultiSelect {
+   border: 1px solid #666;
+   box-shadow: inset 0 0 2px rgba(0,0,0,.3);
+   box-sizing: border-box;
+   padding: 5px 5px 0;
+   position: relative;
+   * {
+      box-sizing: inherit;
+   }
+   &__container {
+   }
+   &__choice {
+      background: #eee linear-gradient(to bottom, #fff, #eee);
+      border: 1px solid #666;
+      border-radius: 3px;
+      cursor: default;
+      display: inline-block;
+      margin: 0 5px 5px 0;
+      padding: 0 20px 0 3px;
+      position: relative;
+      &__content {
+         color: #333;
+         line-height: 22px;
+      }
+      &__close-button {
+         color: #aaa;
+         cursor: pointer;
+         display: block;
+         font-family: @bold-font;
+         font-size: 16px;
+         font-weight: 700;
+         padding: 3px 5px;
+         position: absolute;
+         right: 0;
+         top: 0;
+         &:hover {
+            color: #666;
+         }
+      }
+      &--selected {
+         background: #06a linear-gradient(to bottom, #08c, #06a);
+         border-color: #039;
+         .alfresco-forms-controls-MultiSelect__choice {
+            &__content {
+               color: #fff;
+            }
+            &__close-button {
+               color: #cef;
+               &:hover {
+                  color: #fff;
+               }
+            }
+         }
+      }
+   }
+   &__off-screen-search {
+      font-size: 14px;
+      left: -9999px;
+      margin: 0;
+      padding: 0;
+      position: absolute;
+      top: -9999px;
+   }
+   &__search-box {
+      background: transparent;
+      border: 0;
+      color: #000;
+      display: inline-block;
+      font-size: 14px;
+      line-height: 22px;
+      margin: 0 0 5px 0;
+      max-width: 100%;
+      padding: 0;
+      width: 50px;
+      &:focus {
+         outline: 0;
+      }
+   }
+   &__results {
+      background: #fff;
+      box-shadow: 2px 2px 3px rgba(0, 0, 0, .1);
+      display: none;
+      left: -1px;
+      list-style: none;
+      margin: 0;
+      overflow: hidden;
+      padding: 0;
+      position: absolute;
+      right: -1px;
+      top: 100%;
+      > * {
+         border: solid #666;
+         border-width: 1px 1px 0;
+         font-size: 14px;
+         margin: 0;
+         overflow: hidden;
+         padding: 5px 0 5px 5px;
+         text-overflow: ellipsis;
+         white-space: nowrap;
+      }
+   }
+   &__result-loading, &__result-error, &__result-no-results {
+      border-width: 1px;
+      display: none;
+   }
+   &__result-no-results {
+      color: #999;
```

Comment:
Font colour is almost something we're going to want to be themable via LESS variables